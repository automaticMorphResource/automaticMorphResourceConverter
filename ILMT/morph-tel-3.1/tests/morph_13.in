1	gummadi	unk	
1	wIsukovadaM	unk	
1	valana	unk	
1	cakkeVra	unk	
1	vyAXigraswulaku	unk	
1	rakarakAla	unk	
1	upayogAlu	unk
1	unnAyi	unk	
1	.	unk	
