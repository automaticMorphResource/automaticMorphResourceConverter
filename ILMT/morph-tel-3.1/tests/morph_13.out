1	gummadi	unk	<fs af='gummadi,n,,sg,,d,0,0'>|<fs af='gummadi,n,,sg,,o,ti,ti'>
2	wIsukovadaM	unk	<fs af='wIsuko,v,any,any,any,,adaM,adaM'>|<fs af='wIsuko,v,any,any,any,,adaM,adaM'>
3	valana	unk	<fs af='valana,avy,,,,,0,0_avy'>|<fs af='vala,n,,sg,,,na,na'>
4	cakkeVra	unk	<fs af='cakkeVra,n,,sg,,d,0,0'>|<fs af='cakkeVra,n,,sg,,o,ti,ti'>
5	vyAXigraswulaku	unk	<fs af='vyAXigraswudu,n,,pl,,,ki,ki'>
6	rakarakAla	unk	<fs af='rakarakAlu,n,,pl,,o,ti,ti'>
7	upayogAlu	unk	<fs af='upayogaM,n,,pl,,d,0,0'>|<fs af='upayogaM,n,,pl,,,vu,vu'>
8	unnAyi	unk	<fs af='uMdu,v,n,pl,3,,A,A'>
9	.	unk	<fs af='&dot;,punc,,,,,,'>
